src/pyramid_marrowmailer/tests.py

```diff
@@ -88,14 +89,14 @@ def test_send(self):
         from pyramid_marrowmailer import get_mailer
         mailer = get_mailer(self.request)
 
-	message = mailer.new()
-	message.subject = "foobar"
-	message.to = "foobar@bar.com"
-	message.plain = "hi"
-	message.send()
-	self.assertEqual(self.handler.info, [])
+        message = mailer.new()
+        message.subject = "foobar"
+        message.to = "foobar@bar.com"
+        message.plain = "hi"
+        message.send()
+        self.assertEqual(self.handler.info, [])
+        # self.assertTrue('DELIVER' in self.handler.info[1])
```

Comment:
> Try to run the tests in Python 2.7 and Python 3.4 with the line uncommented so you find out why it is commented out (I really don't remember).

Roger :) 

> Try to accept pull requests more quickly in the future.

I've only seen it now, Github is really bad at notifications (you have to cc someone)